src/App.jsx

```diff
@@ -14,57 +15,70 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGoodу, setSelectedGoodу] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const selected = name => {
+    if (name === selectedGoodу) {
+      setSelectedGoodу('');
+    } else {
+      setSelectedGoodу(name);
+    }
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+  const clear = () => {
+    setSelectedGoodу('');
+  };
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGoodу ? `${selectedGoodу} is` : 'No goods'} selected
+        {selectedGoodу && (
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={clear}
+          />
+        )}
+      </h1>
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+      <table className="table">
+        <tbody>
+          {goods.map(good => {
+            return (
+              <tr
+                data-cy="Good"
+                className={
+                  selectedGoodу === good ? 'has-background-success-light' : ''
+                }
+              >
+                <td>
+                  <button
+                    data-cy={
+                      selectedGoodу === good ? 'RemoveButton' : 'AddButton'
```

Comment:
Create a variable for this check and use it everywhere
```suggestion
                      isSelected ? 'RemoveButton' : 'AddButton'
```